pkg/target/manager.go

```diff
@@ -62,10 +66,14 @@ type Manager interface {
 	// of a pattern
 	TargetMatchPattern(ctx context.Context, value string) error
 
+	// Kubeconfig returns the kubeconfig for the given target cluster
+	Kubeconfig(ctx context.Context, t Target) ([]byte, error)
+	// WriteKubeconfig creates a kubeconfig file in the temporary directory of the os
+	WriteKubeconfig(data []byte) (string, error)
 	// SeedClient controller-runtime client for accessing the configured seed cluster
 	SeedClient(ctx context.Context, t Target) (client.Client, error)
 	// ShootClusterClient controller-runtime client for accessing the configured shoot cluster
```

Comment:
```suggestion
	// ShootClient controller-runtime client for accessing the configured shoot cluster
```

Reply:
I have not changed this line but anyway I have fixed it.